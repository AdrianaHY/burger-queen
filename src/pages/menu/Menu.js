import React, { Component } from 'react';

import Tabs from '../../components/tabs/Tabs';
import List from '../../components/list/List';

import './Menu.scss';

class Menu extends Component {
	render() {
		const {
			breakfastMenu,
			breakfast,
			handlerMenuOption,
			dinnerAndLunchMenu,
			addItem,
			total
		} = this.props;
		return (
			<div className="menu-container">
				<h1 className="title">MENUS</h1>
				<Tabs
					breakfast={breakfast}
					handlerMenuOption={handlerMenuOption}
				/>
				<List
					breakfastMenu={breakfastMenu}
					dinnerAndLunchMenu={ dinnerAndLunchMenu}
					breakfast={breakfast}
					addItem={addItem}
					total={total}
				/>
			</div>
		)
	}
}

export default Menu;