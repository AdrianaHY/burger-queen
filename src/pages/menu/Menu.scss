@import '../../styles/_variables.scss';

.menu-container {
	.title {
		color: $orange;
		font-family: $fontFamily;
		font-size: $title;
	}
}