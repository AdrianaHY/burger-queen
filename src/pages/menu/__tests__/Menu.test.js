import React from 'react';
import {Â shallow } from 'enzyme';
import Menu from '../Menu';

describe('pages | menu | Menu', () => {
	it('Renders the page', () => {
		const props = {
			breakfastMenu: [
				'sandwich',
				'coffe'
			],
			breakfast: true,
			handlerMenuOption: jest.fn(),
			dinnerAndLunchMenu: [
				'simple hamburger',
				'soda'
			],
			addItem: jest.fn(),
			total: 0
		};

		const component = shallow( <Menu {...props} /> );

		expect(component).toMatchSnapshot();
	});
});