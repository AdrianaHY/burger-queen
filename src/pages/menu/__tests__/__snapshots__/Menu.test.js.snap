// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages | menu | Menu Renders the page 1`] = `ShallowWrapper {}`;
